### PR TITLE
benchmark: add tpcc result

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### What is changed, added or deleted? 
add tpcc result

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/benchmark/tpcc.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+使用 BenchmarkSQL 向集群导入 **1000 warehouse** 的数据。通过 HAProxy 代理，分别以递增并发数向集群发送请求，单次并发测试时间 10 分钟。
+
+### TiDB 版本信息
+
+### v3.0.0
+
+| 组件 | GitHash |
+| :-: | :-: |
+| TiDB | 08e56cd3bae166b2af3c2f52354fbc9818717f62 |
```
Comment (jackysp) [inline dev/benchmark/tpcc.md]: need to be updated

File: dev/benchmark/tpcc.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+| :-: | :-: |
+| 172.16.4.75 | 2\*TiDB 2\*TiKV 1\*pd |
+| 172.16.4.76 | 2\*TiDB 2\*TiKV 1\*pd |
+| 172.16.4.77 | 2\*TiDB 2\*TiKV 1\*pd |
+
+## 测试结果
+
+| 版本 | threads | tpmC |
+| :-: | :-: | :-: |
+| v3.0 | 128  | 145102.62 |
```
Comment (jackysp) [inline dev/benchmark/tpcc.md]: ditto

File: dev/benchmark/tpcc.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+| v3.0 | 256  | 161311.9  |
+| v3.0 | 512  | 184991.19 |
+| v2.1 | 128  | 92141.79  |
+| v2.1 | 256  | 107464.93 |
+| v2.1 | 512  | 121350.61 |
+
+![point select](/media/sysbench_v3_point_select.png)
```
Comment (jackysp) [inline dev/benchmark/tpcc.md]: ditto

File: dev/benchmark/tpcc.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+[performance]
+max-procs = 20
+[prepared_plan_cache]
+enabled = true
+```
+
+### TiKV 参数配置
+
+```txt
+[readpool.storage]
```
Comment (jackysp) [inline dev/benchmark/tpcc.md]: ditto

File: dev/benchmark/tpcc.md (hunk 1 above)
Comment (lysu) [inline dev/benchmark/tpcc.md]:
```suggestion
| TiDB | 46c38e15eba43346fb3001280c5034385171ee20 |
```

File: dev/benchmark/tpcc.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+使用 BenchmarkSQL 向集群导入 **1000 warehouse** 的数据。通过 HAProxy 代理，分别以递增并发数向集群发送请求，单次并发测试时间 10 分钟。
+
+### TiDB 版本信息
+
+### v3.0.0
+
+| 组件 | GitHash |
+| :-: | :-: |
+| TiDB | 08e56cd3bae166b2af3c2f52354fbc9818717f62 |
+| TiKV | 57e684016dafb17dc8a6837d30224be66cbc7246 |
```
Comment (lysu) [inline dev/benchmark/tpcc.md]:
```suggestion
| TiKV | a467f410d235fa9c5b3c355e3b620f81d3ac0e0c |
```

File: dev/benchmark/tpcc.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+| PD | 6a7832d2d6e5b2923c79683183e63d030f954563 |
```
Comment (lysu) [inline dev/benchmark/tpcc.md]:
```suggestion
| PD | 70aaa5eee830e21068f1ba2d4c9bae59153e5ca3 |
```

File: dev/benchmark/tpcc.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+```toml
+[log]
+level = "error"
+[performance]
+max-procs = 20
+[prepared_plan_cache]
+enabled = true
+```
+
+### TiKV 参数配置
```
Comment (lysu) [inline dev/benchmark/tpcc.md]: we use default conf for TiKV, so we can remove this part?

**Pull request merged**